op-celestia/indexer/rpc/api.go

```diff
@@ -39,7 +39,7 @@ func GetAPI(api *IndexerAPI) gethrpc.API {
 
 // GetDALocationResponse represents the response for getDALocation
 type GetDALocationResponse struct {
-	Type string      `json:"type"` // "celestia" or "ethereum"
+	Type string      `json:"type"` // "celestia" or "ethereum plain calldata" or "ethereum EIP4844 blobs"
 	Data interface{} `json:"data"` // CelestiaLocation or EthereumLocation
 }
```

Comment:
NOTE: this is a API breaking change, if users depend on `"type: "ethereum"` in responses, we might want to expose separately.

IMHO rather than stringy matching, it would be better to use some kind of `enum` to match strongly against... not sure what is best practice w/ go & JSON responses tbh.

Reply:
can we keep "celestia" "ethereum" and add "eip4844" or similar as a addition to not break existing clients.

Reply:
I see that we're using "ethereum" for both and differentiating via blob_hashes field.